src/mudlet-lua/lua/DebugTools.lua

```diff
@@ -215,6 +215,12 @@ function prettywrite (tbl, space, not_clever)
   return table.concat(lines, #space > 0 and '\n' or '')
 end
 
-function display(what)
-  echo((prettywrite(what, '  ') or 'nil') .. '\n')
+function display(...)
+  if arg.n > 1 then
```

Comment:
`arg` got deprecated in later releases of Lua, while it's not an issue in the immediate future could you make a local table for it anyway?

Reply:
Sure. Just as a side comment, whenever that change comes into Mudlet, it is going to break a lot of old scripts, all by itself.

Reply:
It will... which is why it's not coming soon, we don't break scripts.